packages/core/src/styles/layout/_optimize.scss

```diff
@@ -5,7 +5,8 @@
 // This is a performance optimization that aliases the token to
 // a "minified" value to reduce overall bundle size.
 
-:root {
+:root,
+:host {
```

Comment:
The optimized shorter variables were missing from the base component styles causing styles to fail when global CSS is not used.